xbmc/guilib/GraphicContext.h

```diff
@@ -296,6 +296,6 @@ class CGraphicContext : public CCriticalSection,
  \brief
  */
 
-XBMC_GLOBAL(CGraphicContext,g_graphicsContext);
-
+XBMC_GLOBAL_REF(CGraphicContext,g_graphicsContext);
+#define g_graphicsContext XBMC_GLOBAL_USE(CGraphicContext)
```

Comment:
why don't you kill the useless macro? there is zero advantage in having this macro. imo GlobalsHandling.h is maximum crap and the marco just hides this. make it obvious that it is crap. this rises motivation to get rid of it.